corehq/apps/app_manager/models.py

```diff
@@ -3395,7 +3396,7 @@ def new_module(cls, name, lang):
             case_type='',
             case_details=DetailPair(
                 short=Detail(detail.to_json()),
-                long=Detail(detail.to_json()),
```

Comment:
is this intentional?  (may be fixed in a later commit, flagging to check)

Reply:
fixed later - unfortunately modernize can't tell the difference between ```long``` as a type vs as a kwarg